CITATION.cff

```diff
@@ -0,0 +1,21 @@
+# https://citation-file-format.github.io/
+# https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-citation-files
+
+cff-version: 1.2.0
+title: Standard Architecture for Trusted Research Environments (SATRE)
+message: Please cite SATRE using the metadata from this file
+type: software
+authors:
+  - {}
```

Comment:
What should we put for the authors? The [current Zenodo record](https://zenodo.org/record/8233509) has `Ed Chalstrey; Jim Madge; James Robinson; Simon Li; Arron Lacey; harisood; Matt Craddock; Tim Machin`